mage/changelog.md

```diff
@@ -4,6 +4,87 @@ title: Changelog
 sidebar_label: Changelog
 ---
 
+## v1.4 - Dec 13, 2021
+
+### Major Features and Improvements
+
+- Implemented Link prediction with [DGL](https://www.dgl.ai/).
+  [#160](https://github.com/memgraph/mage/pull/160)
+- Implemented Node classification with PyTorch.
+  [#161](https://github.com/memgraph/mage/pull/161)
+- Added igraph support.
+  [#150](https://github.com/memgraph/mage/pull/150)
+- Added _k_-means embedding clustering algorithm.
+  [#105](https://github.com/memgraph/mage/pull/105)
+- Added better support for C++ API.
+  [#174](https://github.com/memgraph/mage/pull/174)
+- 
+### Bug Fixes
+- Fix bugs in resetting procedures, class labels and early stopping in the Node classification module.
```

Comment:
I would like us to have more user-oriented changelogs..
I won't touch the older versions, but this is the newest one right?
So let's write something here that is comprehensible to the user... so what did you actually fix? What would happen that doesn't happen anymore?

Reply:
Okay makes sense

Reply:
I changed it, so check how it seems to you